logging/cluster-logging-release-notes.adoc

```diff
@@ -100,4 +100,24 @@ As a workaround, if you change the secret, you can force the Fluentd pods to red
 $ oc delete pod -l component=fluentd
 ----
 
+[id="openshift-logging-5-2-0-deprecated-removed-features"]
+== Deprecated and removed features
+
+Some features available in previous releases have been deprecated or removed.
+
+Deprecated functionality is still included in OpenShift Logging and continues to be supported; however, it will be removed in a future release of this product and is not recommended for new deployments.
+
+[id="openshift-logging-5-2-0-legacy-forwarding"]
+=== Forwarding logs using the legacy Fluentd and legacy Syslog methods have been deprecated
+
+From {product-title} version 4.6 to the present, forwarding logs by using the following legacy methods have been deprecated and will be removed in a future release:
```

Comment:
Later in the docs the word "version" is omitted in similar places (it's just OpenShift Container Platform 4.7). Is there maybe a set way of writing this?

Reply:
Updated. Thanks.